python/docs/source/user_guide/pandas_on_spark/transform_apply.rst

```diff
@@ -20,7 +20,7 @@
 Transform and apply a function
 ==============================
 
-.. NOTE: the images are stored at https://github.com/koalas/issues/1443. Feel free to edit and/or add.
+.. NOTE: the images are stored at https://github.com/databricks/koalas/issues/1443. Feel free to edit and/or add.
```

Comment:
On my second thought, maybe we can just remove it?

Reply:
Yes, let's remove it, @itholic . :)

Reply:
cc @HyukjinKwon FYI as an original author.